Update to 1.7.0

Chasethechicken opened the pull request.

Chasethechicken committed:
Update to 1.7.0

TingPing commented on org.mapeditor.Tiled.json:
```diff
@@ -25,20 +25,25 @@
       ],
       "post-install": [
         "sed -i 's/application-x-tiled/org.mapeditor.Tiled-mimetype/g' /app/share/mime/packages/org.mapeditor.Tiled.xml",
+        "sed -i 's/Icon=tiled/Icon=org.mapeditor.Tiled/g' /app/share/applications/org.mapeditor.Tiled.desktop",
```
You can just add a `rename-icon: "tiled"` to the manifest instead of doing this by hand.

TingPing replied:
Also why did the icon name change, did upstream regress?

Chasethechicken replied:
I appreciate your feedback, because I have no experience in packaging for flatpak.
I have fixed the icon renaming.

Chasethechicken replied:
Regarding the icon name, it seems upstream did not change.
But building the current master gives me these warnings:
`Finishing app
Exporting share/applications/org.mapeditor.Tiled.desktop
Exporting share/mime/packages/org.mapeditor.Tiled.xml
Not exporting share/icons/hicolor/32x32/apps/tiled.png, non-allowed export filename
Exporting share/icons/hicolor/32x32/mimetypes/org.mapeditor.Tiled-mimetype.png
Not exporting share/icons/hicolor/16x16/apps/tiled.png, non-allowed export filename
Exporting share/icons/hicolor/16x16/mimetypes/org.mapeditor.Tiled-mimetype.png
Not exporting share/icons/hicolor/scalable/apps/tiled.svg, non-allowed export filename
Exporting share/icons/hicolor/scalable/mimetypes/org.mapeditor.Tiled-mimetype.svg
Not exporting share/icons/hicolor/64x64/apps/tiled.png, non-allowed export filename
Not exporting share/icons/hicolor/128x128/apps/tiled.png, non-allowed export filename
Please review the exported files and the metadata`
And the resulting flatpak does not have an icon.
When updating to 1.7.0 I fixed this, because it gave me a warning.

If you would rather have this as a separate pull request, I could do that as well.

Chasethechicken replied:
I should have read the other issues first :sweat_smile:, I guess this fixes #6 then.

Chasethechicken committed:
Use rename-icon

Flatpak can rename the icon automatically,
which is way cleaner than doing that manually.
